src/envoy/alts/alts_socket.proto

```diff
@@ -0,0 +1,29 @@
+// Copyright 2018 Istio Authors
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+syntax = "proto3";
+
+package envoy.security.v2;
+
+import "google/protobuf/duration.proto";
+import "validate/validate.proto";
+
+message AltsSocket {
```

Comment:
yes, this will be eventually upstream to istio/api or envoyproxy/data-plane-api, in a meanwhile since this is simple enough, a copy or just craft the JSON in pilot should be enough